src/scrabble.js

```diff
@@ -1,5 +1,124 @@
-function scrabble() {
+const alphaGroup = [
+  ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T', 1],
+  ['D', 'G', 2],
+  ['B', 'C', 'M', 'P', 3],
+  ['F', 'H', 'V', 'W', 'Y', 4],
+  ['K', 5],
+  ['J', 'X', 8],
+  ['Q', 'Z', 10]
+]
 
+const scrabble = (word) => {
+  let scrabbleScore = 0
+  if (word === null) return 0
+  const scrabbleWord = word.toUpperCase().split('')
+  let doubleTheWord = []
+  let tripleTheWord = []
+
+  // d{og
+  if (scrabbleWord.includes('{')) {
+    if (scrabbleWord.indexOf('}') === -1) {
+      return scrabbleScore
+    }
+  }
+
+  if (scrabbleWord.includes('}')) {
+    if (scrabbleWord.indexOf('{') === -1) {
+      return scrabbleScore
+    }
+  }
+
+  if (scrabbleWord.includes('[')) {
+    if (scrabbleWord.indexOf(']') === -1) {
+      return scrabbleScore
+    }
+  }
+
+  if (scrabbleWord.includes(']')) {
+    if (scrabbleWord.indexOf('[') === -1) {
+      return scrabbleScore
+    }
+  }
+
+  if (scrabbleWord.includes('{') && scrabbleWord.includes('}')) {
+    const startIndex = scrabbleWord.indexOf('{')
+    const endIndex = scrabbleWord.indexOf('}')
+    doubleTheWord = scrabbleWord.splice(
+      startIndex + 1,
+      endIndex - (startIndex + 1)
+    )
+
```

Comment:
I like the use of splice  but with the second position, would it also work if you just had simply endIndex - 1?